tests/testthat/_snaps/install.md

```diff
@@ -13,3 +19,27 @@
       Consider restarting the R session and loading the newly-installed packages.
       
 
+# install reports on packages about to be installed
+
+    Code
+      install()
+    Output
+      Downloading 1 package(s) and their dependencies
+      Done
+      
+      The following package(s) will be installed into the library
+      - bread     [1.0.0]
+      - breakfast [1.0.0]
+      - oatmeal   [1.0.0]
+      - toast     [1.0.0]
+      
+      Installing oatmeal [1.0.0] ...
```

Comment:
Given that the versions are now reported above, what do you think about dropping from the individual status lines?

Reply:
Dropping the versions, I assume you mean? That sounds good to me.

Reply:
Ok, I'll propose some changes in separate PR.